packages/react-instantsearch-hooks/scripts/transforms/rish-to-ris.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-shadow */
+
+export default function transform(file, api, options) {
+  const j = api.jscodeshift;
+  const printOptions = options.printOptions || { quote: 'single' };
+  const root = j(file.source);
+
+  const replaceImports = (from, to) => (root) =>
+    root
+      .find(j.ImportDeclaration)
+      .filter(
+        (path) =>
+          path.node.source.value === from ||
+          path.node.source.value.startsWith(`${from}/`)
+      )
+      .forEach((sourceImport) => {
+        j(sourceImport).replaceWith(
+          j.importDeclaration(
+            sourceImport.node.specifiers,
+            j.stringLiteral(sourceImport.value.source.value.replace(from, to))
+          )
+        );
+      });
+
+  const replaceHooks = replaceImports(
+    'react-instantsearch-hooks',
+    'react-instantsearch-core'
+  );
+  const replaceHooksWeb = replaceImports(
+    'react-instantsearch-hooks-web',
+    'react-instantsearch'
+  );
+
+  replaceHooks(root);
+  replaceHooksWeb(root);
```

Comment:
Should we also rename react-instantsearch-hooks-server and react-instantsearch-hooks-router-nextjs ?